apps/web/assets/_variables.scss

```diff
@@ -14,7 +14,7 @@ $color-2-secondary-100: 189 238 255;
 $color-2-secondary-200: 128 223 255;
 $color-2-secondary-300: 61 207 255;
 $color-2-secondary-400: 0 191 255;
-$color-2-secondary-500: 0 142 189;
+$color-2-secondary-500: 49 104 125;
```

Comment:
This can't work because we need the whole pallet, so please revert the color changes